### ISSUE TITLE
Auto completion errors

### ISSUE THREAD

**Issue by Nesze** [Homebrew/homebrew-services#549]
I found a minor inconvenience with auto-complete:
`$ brew services -bash: __brewcomp_prev: command not found`

```bash
$ brew --version
Homebrew 3.6.21
Homebrew/homebrew-core (git revision ed10b483eca; last commit 2023-02-12)
Homebrew/homebrew-cask (git revision 2bde4cb0bc; last commit 2023-02-12)
$ bash --version
GNU bash, version 5.2.15(1)-release (aarch64-apple-darwin21.6.0)
```

Think since https://github.com/Homebrew/brew/pull/14545, [__brewcomp_prev](https://github.com/Homebrew/homebrew-services/blob/master/completions/bash/brew-services#L11) is now obsolete.

**Comment by MikeMcQuaid** [comment]
@maxim-belkin mind taking a look? 🙇🏻

**Comment by maxim-belkin** [comment]
> @maxim-belkin mind taking a look? 🙇🏻

Sure. So, here is what's happened: `__brewcomp_prev` function finds previous non-switch word typed by the user on the command line. Currently it is not used by other Bash completion functions in Homebrew/brew, so I removed it in https://github.com/Homebrew/brew/pull/14545. It is, however, used by Bash completions in this repo (Homebrew/services). So, we can either restore it in Homebrew/brew (with a note that it might be used in other repos) or add it to Bash completions here.

**Comment by maxim-belkin** [comment]
@MikeMcQuaid, a gentle ping about the ☝🏻 Our options are: (1) restore __brewcomp_prev in Homebrew/brew or (2) add __brewcomp_prev here.

**Comment by SMillerDev** [comment]
It should be here if this is the only user

**Comment by maxim-belkin** [comment]
@Nesze, thank you for reporting the issue! The fix (#551) has been merged in -- please update homebrew/services, re-source Bash completion script, and let us know if the issue has been resolved.

**Comment by Nesze** [comment]
@maxim-belkin thank you for the quick fix! I can confirm that auto-complete now works w/o any warnings.
